src/components/Trade/TradeInputV2.tsx

```diff
@@ -81,6 +87,18 @@ export const TradeInput = () => {
     selectPortfolioCryptoHumanBalanceByFilter(state, sellAssetBalanceFilter),
   )
 
+  const isSwapperApiPending = useSelector(selectSwapperApiPending)
+  const isTradeQuotePending = useSelector(selectSwapperApiTradeQuotePending)
+  const isUsdRatesPending = useSelector(selectSwapperApiUsdRatesPending)
+
+  const quoteAvailableForCurrentAssetPair = useMemo(() => {
```

Comment:
If `true`, we don't need to show a skeleton over the input fields as we have enough information to estimate all values.

The quote may be stale, but "Preview Trade" is blocked until it refreshes.